src/core/os/amdgpu/dri3/dri3WindowSystem.cpp

```diff
@@ -1116,6 +1116,83 @@ Result Dri3WindowSystem::GetOutputFromConnector(
     return result;
 }
 
+Result Dri3WindowSystem::FindCrtcForOutput(
+    OsDisplayHandle hDisplay,
+    Device*         pDevice,
+    uint32          randrOutput,
+    uint32          rootWindow,
+    uint32*         pRandrCrtc)
+{
+    const Dri3LoaderFuncs& dri3Procs   = pDevice->GetPlatform()->GetDri3Loader().GetProcsTable();
+    xcb_connection_t*const pConnection = dri3Procs.pfnXGetXCBConnection(static_cast<Display*>(hDisplay));
+
+    *pRandrCrtc = 0;
+
+    xcb_randr_get_screen_resources_cookie_t scrResCookie =
+        dri3Procs.pfnXcbRandrGetScreenResources(pConnection, rootWindow);
+
+    xcb_randr_get_screen_resources_reply_t *pScrResReply =
```

Comment:
Asterisk should be next to the type rather than the variable name (e.g., int* pFoo instead of int *pFoo).